src/clj/game/core-actions.clj

```diff
@@ -262,15 +262,16 @@
         (update-ice-strength state side ice)))))
 
 ;;; Runner actions
+
 (defn click-run
   "Click to start a run."
   [state side {:keys [server] :as args}]
   (when (and (not (get-in @state [:runner :register :cannot-run]))
              (can-run-server? state server)
              (can-pay? state :runner "a run" :click 1))
-    (system-msg state :runner (str "makes a run on " server))
-    (run state side server)
-    (pay state :runner nil :click 1)))
+    (let [cost (pay state :runner nil :click 1)]
+      (system-msg state :runner (str (build-spend-msg cost "make a run on") server))
+      (run state side server))))
```

Comment:
Paying cost before triggering the run is likely breaking sundew, as now the click is spent outside a run. Should be fixed by moving the `(run ...)` above the let.